architecture/core_concepts/routes.adoc

```diff
@@ -37,6 +43,29 @@ Configuration] guide for information on deploying a router.
 ====
 endif::[]
 
+A router uses the service selector to find the service and the endpoints backing
+the service. Service-provided load balancing is bypassed and replaced with the
+router's own load balancing. Routers watch the cluster API and automatically
+update their own configuration according to any relevant changes in the API
+objects. Routers may be containerized or virtual. Custom routers can be deployed
+to communicate modifications of API objects to an external routing solution.
+
+In order to reach a router in the first place, requests for host names must
+resolve via DNS to a router or set of routers. The suggested method is to define
+a cloud domain with a wildcard DNS entry pointing to a virtual IP (VIP) backed by
+multiple router instances on designated nodes. Routes can be sharded among the set
+of routers. Router
+VIP configuration is described in
+ifdef::openshift-enterprise,openshift-origin[]
+link:../../admin_guide/high_availability.html#configuring-a-highly-available-routing-service[High Availability].
+endif::[]
+ifdef::openshift-dedicated[]
+the https://docs.openshift.com/enterprise/3.1/admin_guide/high_availability.html#configuring-a-highly-available-routing-service[OpenShift Enterprise Cluster Administration documentation].
+endif::[]
+DNS for addresses
+outside the cloud domain would need to be configured individually. Other
+approaches may be feasible.
```

Comment:
Would it be cleaner to say `If routes use host names/addresses outside the cloud domain, you will need to configure the DNS entries individually`?

The last sentence `other approaches may be feasible` leaves it hanging - can we either add more details or drop it.